prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_AOE_1_20230220-0001.fhir

```diff
@@ -13164,8 +13164,17 @@
               }
             ],
             "name" : {
+              "extension" : [
+                {
+                  "url" : "https://reportstream.cdc.gov/fhir/StructureDefinition/xpn2-given-name",
+                  "valueString" : "GENARO"
```

Comment:
The new mapping relies on the extension to supply the order in which the given names are translated to HL7v2, so the extension needs to be added to maintain the expected output.